src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor

```diff
@@ -1,14 +1,17 @@
 ﻿@using DMPowerTools.Core.Models;
 
-<MudItem xs="1">
-    <MudText Typo="Typo.h6" Color="ModifierColor">@Modifier</MudText>
+<MudItem xs="3" sm="2" md="1">
+    <MudText Typo="Typo.h5" Color="ModifierColor">@Modifier</MudText>
```

Comment:
Does the modifier actually update okay when the RawPoints change?

Reply:
It does, it updates on each arrow click on the MudNumericField, if applicable, or when focus is lost when entering numbers directly into the text portion.

Reply:
neat!